web/templates/page/index.html.eex

```diff
@@ -64,14 +88,27 @@
     </tbody>
   </table>
   <div class="dn-ns tc">
-    <div class="p3">
+    <div class="p3 mt3">
       Sort by: <%= sort_link @conn, @rummage, field: :repo_name, name: "Repo" %>,
       <%= sort_link @conn, @rummage, field: :gh_created_at, name: "Created" %>,
       <%= sort_link @conn, @rummage, field: :gh_updated_at, name: "Updated" %>
     </div>
     <%= for issue <- @issues do %>
-      <%= component("issue_card", issue: issue, labels: get_label_data(issue) ) %>
+      <%= component("issue_card", issue: issue, labels: get_label_data(issue), params: @conn.query_string, label_function: &insert_params/2) %>
     <% end %>
   </div>
 </section>
 <%= pagination_link(@conn, @rummage) %>
+
+<script>
+  var searchOptions = document.getElementById('opts');
+  var filters = document.getElementById('filters');
+
+  filters.className += " dn"
+
+  searchOptions.addEventListener('click', function(e) {
+    e.preventDefault();
+
+    filters.classList.toggle('dn')
```

Comment:
Maybe grab a polyfill for this so it will [work in all browsers](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)

Reply:
Yeah that's a good shout, I'll replace it with className and an if statement :+1: